docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1535 @@
+{
```

Comment:
<p>PyMC</p><p> </p><p>the meaning of  \mu and \sigma is missing</p><p> </p><p>Not sure these formulas are needed for the example.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I'll go ahead and drop them. I have slight trepidation that the discussion below with alpha/beta and kappa might not entirely track without this context, but if it reads okay to you without it, then I'm happy to take it out. </p>